gcsio/src/main/java/com/google/cloud/hadoop/gcsio/GoogleCloudStorageOptions.java

```diff
@@ -91,6 +91,8 @@
   public static final Map<String, String> AUTHORIZATION_HANDLER_PROPERTIES_DEFAULT =
       ImmutableMap.of();
 
+  public static final boolean ENABLE_DEBUG_LOGGING_DEFAULT = false;
```

Comment:
Looks weird as it seems a duplicate of the default value of the `GCS_ENABLE_DEBUG_LOGGING` above?

Reply:
Removed

Reply:
OK. I'm not very familiar with the code style and could be wrong. Just a general observation.